news/2017-01-04-tip4-testing.asciidoc

```diff
@@ -15,16 +15,36 @@ tags: [tips]
 
 toc::[]
 
-In the next article of our Weld Tips series we are going to focus on integration testing of Weld applications.
-By integration testing term we mean testing at some kind of container, which in case of Weld could mean various environments.
-In each case we take advantage of link:http://arquillian.org/[Arquillian] integration testing framework.
-Let's first look at various environments and then look at link:https://github.com/arquillian/arquillian-container-weld[Arquillian Weld Embedded container], which you can use across all the environments.
+In this follow-up article of our Weld Tips series we are going to focus on integration testing of CDI/Weld applications.
+To set things straight - by integration testing we mean testing in some kind of container and hence in certain environment (SE, servlet, full EE server).
+And whatever the environment, we always make use of of link:http://arquillian.org/[Arquillian] integration testing framework. 
+
+== Arquillian as the chosen framework for testing
+While Arquillian may look complex at first, it is certainly worth it. Here are some of the advantages it gives you:
+
+* Allows usage of `@Inject` directly in test classes
```

Comment:
Ok then it's reasonable to mention. Could be or we can say only CDI injection or something like that. 
Maybe we could also add some warning about client test mode.